modules/ROOT/pages/_partials/task-add-api-instance.adoc

```diff
@@ -143,6 +151,9 @@ This option is not available for Flex Gateway runtime at this time.
 NOTE: Anypoint API Manager supports OpenAPI Specification (OAS) 3.0, with the exception of the callback feature. To work around this issue, handle the callback outside of the Mule runtime engine domain or use an OAS 3.0 specification that does not use callbacks.
 
 .. Optionally, click *Advanced* to edit the *AssetId* and the *Version*.
+.. View the *Conformance Status* of the API to ensure the API is conformant. However, you must fix some conformance issues, such as applying policies, after deploying the API instance.
```

Comment:
I suggest editing this similarly to the prior entry. For example:

If the API is deployed, ...